airbrakes/data_handling/apogee_predictor.py

```diff
@@ -152,10 +149,15 @@ def _get_apogee(self, params: npt.NDArray[np.float64]) -> np.float64:
         if params is None:
             return 0.0
```

Comment:
why do we return 0? I know its like convention or whatever to return 0 to show it ended without problems. At the very least if we are returning something, add it to the docstring and the return type

Reply:
I don't know if there ever is a case where we did return 0.0. The coverage would help verify that actually. So i'll wait for that to be merged.

Reply:
wait disregard this one, I'm tweaking.

the :return in docstring should probably still specify it returns 0.0 if there was a problem/no apogee predicted